examples/basic.py

```diff
@@ -50,8 +49,7 @@ async def help_request(msg):
     # and trigger timeout if not faster than 500 ms.
     try:
         response = await nc.request("help", b'help me', timeout=0.5)
-        print("Received response: {message}".format(
-            message=response.data.decode()))
+        print("Received response: {response.data.decode()}")
```

Comment:
This string is missing the `f` prefix

Reply:
My bad. Fixed with baf159b4de178a2479045badf2771b85daa740ec.